v8env/src/fly/cache/index.ts

```diff
@@ -20,9 +20,10 @@
 /** */
 declare var bridge: any
 export interface CacheSetOptions {
-  ttl?: number,
-  tags?: string[],
-  onlyIfEmpty?: boolean
+  ttl?: number;
+  tags?: string[];
+  onlyIfEmpty?: boolean;
+  skipCacheHeaders?: string[];
```

Comment:
The `skipCacheHeaders` only applies to the response cache, so probably better to make a new `ResponseCacheSetOptions` for this purpose.